compiler/rustc_session/src/config.rs

```diff
@@ -1217,6 +1218,7 @@ fn default_configuration(sess: &Session) -> CrateConfig {
     ret.insert((sym::target_pointer_width, Some(Symbol::intern(&wordsz))));
     ret.insert((sym::target_env, Some(Symbol::intern(env))));
     ret.insert((sym::target_abi, Some(Symbol::intern(abi))));
+    ret.insert((sym::relocation_model, Some(relocation_model)));
```

Comment:
This should be gated behind `sess.is_nightly_build()` to avoid immediately stabilizing it.

Reply:
Done and added a test to make sure as well.